tests/Nox.Generator.Tests/files/yaml/design/api/Entities/country.entity.nox.yaml

```diff
@@ -129,4 +129,8 @@ persistence:
         raiseEvents: domainAndIntegrationEvents
     delete:
         isEnabled: true
-        raiseEvents: domainAndIntegrationEvents
\ No newline at end of file
+        raiseEvents: domainAndIntegrationEvents
+        
+events:
+   - name: TestEvent
```

Comment:
Why this event is not generated? Please use some better names for the examples.

Reply:
This test does not generate file, and this lines have been removed.